tests/test_FiberFlat.py

```diff
@@ -0,0 +1,67 @@
+import sys
+import unittest
+import pickle
+
+import numpy as np
+
+import lsst.utils.tests
+import lsst.pex.exceptions
+import lsst.geom
+import lsst.afw.image
+import lsst.afw.image.testUtils
+
+import pfs.drp.stella as drpStella
+import lsst.afw.image as afwImage
+from pfs.drp.stella.constructFiberFlatTask import ConstructFiberFlatTask
+
+
+class FiberFlatTestCase(lsst.utils.tests.TestCase):
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    def testExtractNonOptimal(self):
+        """ Tests the ConstructFiberFlatTask.correctLowVarianceImage method
+        """
+        width = 4096
+        height = 4176
+
+        defaultVarValue = 10.0
+        defaultImageValue = 1.0
+        defaultMaskValue = 0x0
+        minVariance = 4.0
+
+        affectedPixelRow = 123
+        affectedPixelCol = 456
+
+        maskedImage = afwImage.MaskedImageF(width, height)  # Image for extraction
+        maskedImage.getImage().getArray()[:] = defaultImageValue
+        maskedImage.getMask().getArray()[:] = defaultMaskValue
+        maskedImage.getVariance().getArray()[:] = defaultVarValue
+        maskedImage.getVariance().getArray()[affectedPixelRow, affectedPixelCol] = defaultImageValue
```

Comment:
`maskedImage.getVariance().getArray()` --> `maskedImage.variance.array`, etc.

Here and below.

Reply:
And then you don't need the `[:]` either; `maskedImage.variance.array = defaultVarValue`.  Or I think `maskedImage.variance[:] = defaultVarValue` will work (but you'd have to check)

Reply:
As discussed on slack, keeping ``[:] = `` style assignments.